public/src/pages/auth/Login/Login.tsx

```diff
@@ -74,9 +76,16 @@ export const Login: FC = () => {
         setLoginResponse(data);
         return data;
       })
-      .then(({ email, errorText }) => {
-        setErrorText(errorText);
+      .then(async ({ email, errorText }) => {
+        errorText ?? setErrorText(errorText);
```

Comment:
I suggest using if statements instead. Avoid using null coalescing (`??`) or `||` operator in such a way

Reply:
Sure, just out of personal interest, is it due to readability or something else?